pkg/orm/orm.go

```diff
@@ -64,18 +61,7 @@ type Orm struct {
 	config *OrmConfig
 }
 
-// Migration represents a database migration (a modification to be made on the database).
-type Migration struct {
-	// ID is the migration identifier. Usually a timestamp like "201601021504".
-	ID string
-	// Migrate is a function that will be executed while running this migration.
-	Migrate gormigrate.MigrateFunc
-	// Rollback will be executed on rollback. Can be nil.
-	Rollback gormigrate.RollbackFunc
-}
-
 // NewMySqlOrm - creates a new Orm object with MySQL connection
-// TODO: rename to NewMySQLOrm
 func NewMySqlOrm(config *OrmConfig) *Orm {
```

Comment:
`func NewMySQLOrm ...`
It should be renamed to be more consistent with golint rules...

Reply:
That would be a breaking change